bld/cb.cs

```diff
@@ -1775,7 +1775,7 @@ static void write_e_sqlcipher()
 			var defines = new Dictionary<string,string>
 			{
 				{ "_WIN32", null }, // for tomcrypt
-				{ "ENDIAN_LITTLE", null }, // for tomcrypt arm
+				{ "ENDIAN_LITTLE", "" }, // for tomcrypt arm
```

Comment:
The definition change is here only to silence warnings (redefinition is ignored if value is identical).